tensorflow/inference/docker/2.0.2/py2/Dockerfile.cpu

```diff
@@ -0,0 +1,84 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    curl \
+    git \
+    wget \
+    vim \
+ && rm -rf /var/lib/apt/lists/*
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+
+# Add arguments to achieve the version, python and url
+#  PYTHON=python for 2.7
+#  PYTHON=python3 for 3.5, 3.6 is not available directly on 16.04
+ARG PYTHON=python
+
+# user python-pip or python3-pip
+ARG PYTHON_PIP=python-pip
+
+#  PIP=pip for 2.7
+#  PIP=pip3 for 3.5, 3.6 is not available directly on 16.04
+ARG PIP=pip
+
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ${PYTHON} \
+    ${PYTHON_PIP}
+
+RUN ${PIP} --no-cache-dir install --upgrade \
+    pip \
+    setuptools
+
+# Some TF tools expect a "python" binary
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python
+
+RUN curl https://tensorflow-aws.s3-us-west-2.amazonaws.com/MKL-Libraries/libiomp5.so -o /usr/local/lib/libiomp5.so
+RUN curl https://tensorflow-aws.s3-us-west-2.amazonaws.com/MKL-Libraries/libmklml_intel.so -o /usr/local/lib/libmklml_intel.so
+
+ENV LD_LIBRARY_PATH '/usr/local/lib:$LD_LIBRARY_PATH'
+
+RUN curl https://tensorflow-aws.s3-us-west-2.amazonaws.com/2.0.2/Serving/CPU-WITH-MKL/tensorflow_model_server -o /usr/bin/tensorflow_model_server \
+ && chmod 555 /usr/bin/tensorflow_model_server
+
+RUN ${PIP} install -U --no-cache-dir \
+    docutils==0.15.2 \
+    awscli \
+    requests==2.22.0
```

Comment:
We need pyYAML v.5.3.1 in this dockerfile and in this layer.

Reply:
Same for GPU py2 Inference

Reply:
added